### PR TITLE
Updated package version and changelog

### PR DESCRIPTION
**Pull request by surajeaton**
Fixes # BLUI-5996

#### Changes proposed in this Pull Request:

- Publish beta for okta login screen

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/CHANGELOG.md [hunk 1]
```diff
@@ -5,11 +5,12 @@ All notable changes to this project will be documented in this file.
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v4.0.4 (Unreleased)
+## v4.1.0 (Unreleased)
 
 ### Added
 
--   Error Manager for ChangePasswordDialog ([#612](https://github.com/etn-ccis/blui-react-workflows/issues/612))
+-   Reusable okta login screen.
+-   Error Manager for ChangePasswordDialog ([#612](https://github.com/etn-ccis/blui-react-workflows/issues/612)).
```
Comment (ektaghag-eaton) [inline login-workflow/CHANGELOG.md:14]: @KomalGDhere-eaton , can you please make sure all issues that we fixed are logged here.

Comment (KomalGDhere-eaton) [reply login-workflow/CHANGELOG.md:20]: sure

File: login-workflow/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@brightlayer-ui/react-auth-workflow",
-    "version": "4.0.3",
+    "version": "4.1.0-beta.0",
```
Comment (ektaghag-eaton) [inline login-workflow/package.json:3]: package version should be 5.0.0-beta.0.

File: login-workflow/CHANGELOG.md [hunk 3]
```diff
@@ -5,11 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v4.0.4 (Unreleased)
+## v4.1.0 (Unreleased)
```
Comment (ektaghag-eaton) [inline login-workflow/CHANGELOG.md:8]: 5.0.0 (Unreleased)

**Review by ektaghag-eaton — APPROVED** [review]

File: login-workflow/CHANGELOG.md [hunk 4]
```diff
@@ -5,11 +5,18 @@ All notable changes to this project will be documented in this file.
...
 
--   Error Manager for ChangePasswordDialog ([#612](https://github.com/etn-ccis/blui-react-workflows/issues/612))
+-   Reusable okta login screen.
+-   Error Manager for ChangePasswordDialog ([#612](https://github.com/etn-ccis/blui-react-workflows/issues/612)).
+-   Reload EULA should not be tied to checkbox status ([#549](https://github.com/etn-ccis/blui-react-workflows/issues/549)).
+-   Eliminate `WorkflowCardFinishState` and introduce `EmptyStateProps` in the success screen. ([#552](https://github.com/etn-ccis/blui-react-workflows/issues/552)).
+-   Updated password validation callaback function.([#560]https://github.com/etn-ccis/blui-react-workflows/issues/560).
+-   SX Style overrides on full screens([#599](https://github.com/etn-ccis/blui-react-workflows/issues/599)).
+-   Added BLUI class names for ChangePasswordDialog ([#600](https://github.com/etn-ccis/blui-react-workflows/issues/600)).
+
```
Comment (joebochill) [inline login-workflow/CHANGELOG.md:19]: These should not all be under the 'Added' category. Some are 'Changed' 'Removed' or 'Fixed'. Refer to https://keepachangelog.com/en/1.1.0/

Comment (ektaghag-eaton) [reply login-workflow/CHANGELOG.md:19]: > These should not all be under the 'Added' category. Some are 'Changed' 'Removed' or 'Fixed'. Refer to https://keepachangelog.com/en/1.1.0/

Oh yes! I missed this, sorry.

File: login-workflow/CHANGELOG.md [hunk 5]
```diff
@@ -5,11 +5,21 @@ All notable changes to this project will be documented in this file.
...
+-   Error Manager for ChangePasswordDialog ([#612](https://github.com/etn-ccis/blui-react-workflows/issues/612)). 
+-   Added BLUI class names for ChangePasswordDialog ([#600](https://github.com/etn-ccis/blui-react-workflows/issues/600)).
+
+### Changed 
+
+-  Reload EULA should not be tied to checkbox status ([#549](https://github.com/etn-ccis/blui-react-workflows/issues/549)).
+-  Eliminate `WorkflowCardFinishState` and introduce `EmptyStateProps` in the success screen([#552](https://github.com/etn-ccis/blui-react-workflows/issues/552)).
+-  Updated password validation callaback function ([#560]https://github.com/etn-ccis/blui-react-workflows/issues/560).
+-  SX Style overrides on full screens ([#599](https://github.com/etn-ccis/blui-react-workflows/issues/599)).
+
```
Comment (joebochill) [inline login-workflow/CHANGELOG.md:22]:
```suggestion
-   Okta redirect-compatible login screen. 
-   Error Manager for ChangePasswordDialog ([#612](https://github.com/etn-ccis/blui-react-workflows/issues/612)). 
-   BLUI class names for ChangePasswordDialog ([#600](https://github.com/etn-ccis/blui-react-workflows/issues/600)).
-  SX style overrides on full screens ([#599](https://github.com/etn-ccis/blui-react-workflows/issues/599)).

### Fixed 

-  Reload EULA should not be tied to checkbox status ([#549](https://github.com/etn-ccis/blui-react-workflows/issues/549)).
-  Updated password validation callback function ([#560]https://github.com/etn-ccis/blui-react-workflows/issues/560).

### Changed

-  Eliminate `WorkflowCardFinishState` and introduce `EmptyStateProps` in the success screen([#552](https://github.com/etn-ccis/blui-react-workflows/issues/552)).

```

**Review by joebochill — APPROVED** [review]

**Pull request merged**